go.sum

```diff
@@ -1,3 +1,19 @@
+cloud.google.com/go v0.26.0/go.mod h1:aQUYkXzVsufM+DwF1aE+0xfcU+56JwCaLick0ClmMTw=
```

Comment:
why is so much added to go.sum? I would have expected more edits or replacements.

Reply:
`Viper` and `logrus` got updated for some reason. I rolled back these changes and I'll put up a separate PR to make those updates. Both `go.mod` and `go.sum` have been cleared and only differ in the `aerospike` packages which is what thisPR is about